gripper_test_controller:
  controller:
    package: bender_arm_control
    module: gripper_controller
    type: GripperActionController
  gripper_action_node:
    goal_tolerance: 0.15
    max_effort: 950
    effort_tolerance: 100
    stall_timeout: 1.0
    stalled_velocity_threshold: 0.1
    state_update_rate: 30